actions/ckeditor/upload.php

```diff
@@ -28,7 +28,7 @@
 
 $file = new ElggFile();
 $file->owner_guid = $user->guid;
-$file->setFilename("ckeditor/{$hash}.jpg");
+$file->setFilename("ckeditor/$hash");
```

Comment:
uploaded file extension is reserved, just not showing, when you click the image to save to your local device, you will see the extension, so no needs to add the jpg